index.html

```diff
@@ -0,0 +1,273 @@
+                    <ul class="d-flex space-x-1 list-tags no-pad">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="btn-action" type="button">See project</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="card grid">
+                <div class="">
+                    <img src="./images/project3.png" class="image-auto-h" alt="project">
+                </div>
+                <div>
+                    <h1 class="m0 font-32">Multi-Post Stories</h1>
+                    <ul class="d-flex space-x-1 p-highlight">
+                        <li class="font-13">Facebook</li>
+                        <li>Full Stack</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        Experimental content creation feature 
+                        that allows users to add to an 
+                        existing story over the course of a day
+                         without spamming their friends.
+                    </p>
+                    <ul class="d-flex space-x-1 list-tags no-pad">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="btn-action" type="button">See project</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="card grid">
+                <div>
+                    <img src="./images/project4.png" class="image-auto-h" alt="project">
+                </div>
+                <div>
+                    <h1 class="m0 font-32">Multi-Post Stories</h1>
+                    <ul class="d-flex space-x-1 p-highlight">
+                        <li class="font-13">Facebook</li>
+                        <li>Full Stack</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        Experimental content creation feature 
+                        that allows users to add to an 
+                        existing story over the course of a day
+                         without spamming their friends.
+                    </p>
+                    <ul class="d-flex space-x-1 list-tags no-pad">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="btn-action" type="button">See project</button>
+                </div>
+            </div>
+        </section>
+    </div>
+    <!-- end of projects section -->
+    
+    <!-- about and contact section -->
+    <div class="about-contact">
+        <section class="about-me d-flex flex-column">
+            <div class="personal">
+                <h1>About <br>Abiola</h1>
```

Comment:
- [ ] Here also, check   line `97` on a detailed review on why you need just one h1 element in a project 👍